sign_language_datasets/datasets/signsuisse/signsuisse.py

```diff
@@ -0,0 +1,172 @@
+"""A Swiss Sign Language Lexicon, combines all three Swiss sign languages: the German-Swiss Sign Language (DSGS), the Langue des Signes Française (LSF) and the Lingua Italiana dei Segni (LIS). ."""
+import json
+import re
+import string
+
+import tensorflow_datasets as tfds
+
+from ..warning import dataset_warning
+from ...datasets import SignDatasetConfig
+
+_DESCRIPTION = """
+Waren Sie schon immer neugierig darauf, wie die Gebärdensprache funktioniert? Folgen Sie einem Gebärdensprache-Kurs und möchten Ihren Wortschatz ausbauen? Oder sind Sie selber gehörlos und stellen fest, dass Ihnen eine Gebärde fehlt?
+
+Genau dafür ist unsere Website da! Hier finden Sie ein in seiner Art einmaliges Lexikon, das alle drei Schweizer Gebärdensprachen zusammenfasst: die Deutschschweizerische Gebärdensprache (DSGS), die Langue des Signes Française (LSF) und die Lingua Italiana dei Segni (LIS). 
+
+Um gleich loszulegen, geben Sie einfach den gesuchten Begriff in das obige Feld ein.
+"""
+
+# TODO(signsuisse): BibTeX citation
+_CITATION = """
+"""
+
+SITE_URL = "https://signsuisse.sgb-fss.ch"
+
+
+class SignSuisse(tfds.core.GeneratorBasedBuilder):
+    """DatasetBuilder for signsuisse dataset."""
+
+    VERSION = tfds.core.Version("1.0.0")
+    RELEASE_NOTES = {
+        "1.0.0": "Initial crawl.",
+    }
+
+    BUILDER_CONFIGS = [
+        SignDatasetConfig(name="default", include_video=True),
+        SignDatasetConfig(name="annotations", include_video=False),
+    ]
+
+    def _info(self) -> tfds.core.DatasetInfo:
+        """Returns the dataset metadata."""
+
+        features = {
+            "id": tfds.features.Text(),
+            "name": tfds.features.Text(),
+            "category": tfds.features.Text(),
+            "spokenLanguage": tfds.features.Text(),
+            "signedLanguage": tfds.features.Text(),
+            "url": tfds.features.Text(),
+            "paraphrase": tfds.features.Text(),
+            "definition": tfds.features.Text(),
+            "exampleText": tfds.features.Text(),
+            "exampleVideo": tfds.features.Text(),
+        }
+
+        if self._builder_config.include_video and self._builder_config.process_video:
+            features["video"] = self._builder_config.video_feature((640, 480))
+        else:
+            features["video"] = tfds.features.Text()
+
+        return tfds.core.DatasetInfo(
+            builder=self,
+            description=_DESCRIPTION,
+            features=tfds.features.FeaturesDict(features),
+            homepage=SITE_URL,
+            supervised_keys=None,
+            citation=_CITATION,
+        )
+
+    def _list_all_lexicon_items(self, dl_manager: tfds.download.DownloadManager):
+        # The lexicon does not allow free search. One must search at least two letters.
+        letters = [c1 + c2 for c1 in string.ascii_lowercase for c2 in string.ascii_lowercase]
```

Comment:
I believe that this will miss character combinations with umlauts. Example: the sign "Ära" (era) from this request:

https://signsuisse.sgb-fss.ch/index.php?eID=signsuisse_search&sword=%C3%A4r&lang=all&category=pleaseselect&curlang=de

Reply:
... just adding `["ä", "ü", "ö"]` to the list should do the trick

Reply:
verified, it is not the case - searching`ar` (https://signsuisse.sgb-fss.ch/index.php?eID=signsuisse_search&sword=ar&lang=all&category=pleaseselect&curlang=de) results in `ÄRA` as the first result

Reply:
thanks for checking!